test/dlc_tests/eks/mxnet/training/test_eks_mxnet_multinode_training.py

```diff
@@ -17,7 +17,7 @@
 LOGGER = eks_utils.LOGGER
 
 
-@pytest.mark.skipif(is_pr_context(), reason=SKIP_PR_REASON)
+#@pytest.mark.skipif(is_pr_context(), reason=SKIP_PR_REASON)
```

Comment:
To be reverted before PR merge

Reply:
probably we could add an eks multinode option in the developer config file. Out of scope for this PR

Reply:
Surely, we should do that. Can we make that a future task? Just wanting to avoid delays for now.

Reply:
EKS test infra is not designed to run on PR context. We might have to add extra configuration other than dev config file